tests/fsharp/core/patterns/test.fsx

```diff
@@ -910,7 +910,7 @@ module AndrewKennedyFunkyActivePatternsBugRelatedToArityInference =
 
     let (Fun(f)) = (fun x -> x)
 
-module AndrewKennedyRatherEmarrassingSimpleBug1133 = 
+module AndrewKennedyRatherEmbarrassingSimpleBug1133 = 
```

Comment:
One day we should do a run through the weird test names

Reply:
Don't think it's worth the time. It will also break historical view in CI.